patches/chromium/.patches

```diff
@@ -103,7 +103,6 @@ fix_use_delegated_generic_capturer_when_available.patch
 expose_webblob_path_to_allow_embedders_to_get_file_paths.patch
 fix_move_autopipsettingshelper_behind_branding_buildflag.patch
 revert_remove_the_allowaggressivethrottlingwithwebsocket_feature.patch
-fix_activate_background_material_on_windows.patch
```

Comment:
Did you intend to delete this? if yes, you also need to delete the patch file itself

Reply:
Ah, I might have missed this part during the rebase.😓